controllers/movies-class.ts

```diff
@@ -0,0 +1,184 @@
+import path from "path";
+import { Like } from "typeorm";
+import { Result, ValidationError } from "express-validator";
+import { calculateTotalTime, calculateTotalVotes, filterCodes, filterImdbUrls, makeTinyUrl, readMovies, MovieQuery } from "../util";
+import { Movie } from "../models/movie";
+
+export class Movies {
+
+    async getAllMovies(queries: MovieQuery) {
+        try {
+            const { file, actor, imdbSort, year } = queries;
+            let movies;
+    
+            if (file === "true") {
+                const data = await readMovies(path.join(__dirname, '../movies.json'));
+                return data;
+            }
+            if (year) {
+                movies = await Movie.find({
+                    order: {
+                        Year: year === 'ASC' ? 'ASC' : 'DESC',
+                    }
+                });
+            }
+            else if (imdbSort) {
+                movies = await Movie.find({
+                    order: {
+                        imdbRating: imdbSort === 'ASC' ? 'ASC' : 'DESC',
+                    },
+                    select: {
+                        Title: true,
+                        imdbID: true
+                    }
+                });
+            }
+            else if (actor) {
+                movies = await Movie.find({
+                    where: {
+                        Actors: Like(`%${actor}%`)
+                    }
+                })
+            }
+            else {
+                movies = await Movie.find();
+            }
+            return movies;
+        }
+        catch (err: any) {
+            throw new Error(err);
+        }
+    }
+
+    async getMoviesData() {
+        try {
+            const movies = await Movie.find();
+            const total = calculateTotalTime(movies);
+            const imdbUrls = filterImdbUrls(movies);
+            const totalVotes = calculateTotalVotes(movies);
+            const codes = filterCodes(movies);
+            const myUrl = await makeTinyUrl(movies);
+    
+            return {
+                totalLengthOfAllMovies: total + " min",
+                imdbUrls: imdbUrls, totalVotes: totalVotes,
```

Comment:
Define the parameters in separate rows, 
```suggestion
                imdbUrls,
                totalVotes,
```